recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
-{% set version = "0.7.0" %}
+{% set version = "0.8.0" %}
 
 package:
   name: pyorc
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pyorc/pyorc-{{ version }}.tar.gz
-  sha256: 052a399558ef9fff1c1607ca9c487778401a9e0a53885814ff0725290b856ab0
+  sha256: 3f52243b7e300cb0fbfb7621d30693cf21392ec5ead3bb32a31d44f2d6bf656a
 
 build:
   number: 0
-  script: PYORC_SKIP_ORC_BUILD=1 {{ PYTHON }} -m pip install . -vv
+  script: PYORC_SKIP_ORC_BUILD=1 PYORC_LIBRARIES="orc" {{ PYTHON }} -m pip install . -vv
```

Comment:
Since we only need to link to `orc` when ORC is prebuilt (as is the case here), we just override the libraries linked to, which seems to resolve the build issue seen previously [on CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=623060&view=logs&j=bb1c2637-64c6-57bd-9ea6-93823b2df951&t=350df31b-3291-5209-0bb7-031395f0baa1 ) ([attached log]( https://github.com/conda-forge/pyorc-feedstock/files/10179224/pyorc-feedstock_20221204.1_linux_64_python3.9.____cpython.txt ) for posterity)